validator/sawtooth_validator/journal/consensus/dev_mode/dev_mode_consensus.py

```diff
@@ -32,11 +32,12 @@ class BlockPublisher(BlockPublisherInterface):
     other consensus algorithms
 
     """
-    def __init__(self, block_cache, state_view):
-        super().__init__(block_cache, state_view)
+    def __init__(self, block_cache, state_view, transaction_sender):
```

Comment:
There are some other consensus impl's in journal/consensus that probably need this change given the change to the interface.